stripe/src/main/java/com/stripe/android/net/TelemetryClientUtil.java

```diff
@@ -0,0 +1,132 @@
+package com.stripe.android.net;
+
+import android.content.Context;
+import android.content.pm.PackageInfo;
+import android.content.pm.PackageManager;
+import android.os.Build;
+import android.provider.Settings;
+import android.support.annotation.NonNull;
+
+import com.stripe.android.BuildConfig;
+import com.stripe.android.util.StripeTextUtils;
+
+import java.math.BigDecimal;
+import java.math.MathContext;
+import java.util.HashMap;
+import java.util.Locale;
+import java.util.Map;
+import java.util.TimeZone;
+import java.util.concurrent.TimeUnit;
+
+class TelemetryClientUtil {
+
+    static Map<String, Object> createTelemetryMap(@NonNull final Context context) {
+        Map<String, Object> telemetryMap = new HashMap<>();
+        Map<String, Object> firstMap = new HashMap<>();
+        Map<String, Object> secondMap = new HashMap<>();
+        telemetryMap.put("v2", 1);
+        telemetryMap.put("tag", BuildConfig.VERSION_NAME);
+        telemetryMap.put("src", "android-sdk");
+
+        firstMap.put("c", createSingleValuePair(Locale.getDefault().toString()));
+        firstMap.put("d", createSingleValuePair(getAndroidVersionString()));
```

Comment:
@bg-stripe that would be great. We can keep the concatted string and you can put the platform under the `r` key and the model under the `s` key.